### PR TITLE
Added Xml writing implicits and convenience method

### PR DESCRIPTION
**Pull request by rcaloras**

### CODE REVIEW

File: src/main/scala/com/gilt/fedex/Implicits.scala [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package com.gilt.fedex
+
+import com.fedex.ship.stub.RequestedShipment
+
+object Implicits {
+
+  implicit def requestedShipmentToXmlType[A](rs: RequestedShipment) =  {
+   new XmlRequestedShipment(rs)
```
Comment (kiambogo) [inline src/main/scala/com/gilt/fedex/Implicits.scala]: 2 spaces?

Comment (rcaloras) [inline src/main/scala/com/gilt/fedex/Implicits.scala]: Done.

**Comment by aweistgilt** [comment]
lgtm

**Comment by kiambogo** [comment]
I'd say that the test cases have the potential to be more comprehensive (than just checking to see if the toXML doesn't return an empty string), but you'd probably be analyzing the output of the string to add to the tests so it's a bit redundant; not sure if that's necessary then.

LGTM 👍

**Comment by rcaloras** [comment]
@kiambogo you're right. I updated the tests to at least check the inclusion of specific xml elements. A slight improvement to checking the empty string.

**Comment by rcaloras** [comment]
Merging
👍 🍔 🍟

**Pull request merged**